source/common/config/well_known_names.h

```diff
@@ -0,0 +1,96 @@
+#pragma once
```

Comment:
Thanks for adding this. Can you merge https://github.com/lyft/envoy/blob/master/source/common/config/metadata.h#L43 into this as well?